spring-boot-modules/spring-boot-crud/src/main/java/com/baeldung/entity_dto_differences/Application.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.entity_dto_differences;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+
```

Comment:
Can we please drop these extra blank lines at the start of methods, and same thing in all the other classes. If you use the standard Baeldung formatter, I believe it should fix code style issues like this automatically, to match the Baeldung guidelines.

Reply:
I'm using the formatter, but it doesn't delete these newlines. I've removed them manually